JASP-Engine/JASP/po/R-nl.po

```diff
@@ -1073,7 +1080,7 @@ msgid ""
 "the Inherent \n"
```

Comment:
Moeten hier de newlines niet ook uit worden gehaald? Hoewel dat eigk meer een ding is om in de analyse zelf te doen.
@koenderks misschien is dit nog wel iets om voor de release naar te kijken. Nu die vaste grootte erin zit zijn al jouw handgeplaatste newlines overbodig geworden.

Reply:
@koenderks en @JorisGoosen 
Ik zal hier nog een apart Issue van maken om al die newlines goed te krijgen en even allemaal langs te lopen, vnl in Audit.